microsoft-edge/devtools-guide-chromium/workspaces/index.md

```diff
@@ -22,145 +22,168 @@ ms.date: 05/04/2021
    limitations under the License.  -->
 # Edit files with Workspaces (Filesystem tab)
 
```

Comment:
```suggestion

Use the **Filesystem** tab in the **Sources** tool to define a workspace, to save DevTools changes in your source code files rather than only in a transient copy of the files that are returned by the web server.

```